core/services/cable_guy/api/manager.py

```diff
@@ -302,6 +302,10 @@ def get_interfaces(self) -> List[EthernetInterface]:
         """
         result = []
         for interface, addresses in psutil.net_if_addrs().items():
+            # We don't care about virtual ethernet interfaces
+            if str(interface).startswith("veth"):
```

Comment:
The comment is ok, but it does not explain the reason of 'why', and that may be not clear to someone reading the code
```suggestion
            # Ignore virtual ethernet interfaces
            ## Virtual interfaces are created by programs such as docker
            ## and they are an abstract of real interfaces, the ones that we want to configure
```

Reply:
Better. Changing.